build/PackageInfo.xml

```diff
@@ -0,0 +1,6 @@
+<pkg-info format-version="2" identifier="io.lbry.LBRY.pkg" version="0.10.0" install-location="/" auth="root">
+  <payload installKBytes="173006" numberOfFiles="3608"/>
```

Comment:
find root | wc -l < if executed in the .pkg directory will show the number of lines and

du -b -s root < will give the number of bytes (needs to be converted to kbytes)